rest-backend/src/main/java/com/sap/psr/vulas/backend/rest/HubIntegrationController.java

```diff
@@ -571,19 +571,19 @@ private VulnerableItemDependency(String _spaceToken, Application _app, Vulnerabl
 			// Scope excluded: State to 1 (False Positive: Secure by design)
 			// Bug excluded: State to 4 (False Positive: Sufficient mitigation in place)
 			// Else: State to 2 (True Positive)
-			final Excemption exc = _vd.getExcemption();
-			if(exc!=null && exc.isExcludedBug()) {
+			final IExemption exc = _vd.getExemption();
+			if(exc!=null) {
 				this.status = STATUS_AUDITED;
+				this.exemptionReason = exc.getReason();
+				
+				// ExemptionBug
 				this.state = FP_MITIGATED;
-				this.exemptionReason = exc.getExcludedBugReason();
-			}
-			else if(exc!=null && exc.isExcludedScope()) {
-				this.status = STATUS_AUDITED;
-				this.state = FP_SECURE_DESIGN;
```

Comment:
I do not remember if scope exemption were treated different in terms of deciding which value to assign to the Vulas finding in SVM (i.e., state FP_SECURE_DESIGN vs FP_MITIGATED)

Reply:
Yes, and I just re-established the old logic